com.djoffe.gnukem.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id": "com.djoffe.gnukem",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "davegnukem",
+  "rename-desktop-file": "davegnukem.desktop",
+  "rename-icon-file": "davegnukem.png",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=network",
+    "--persist=.gnukem"
+  ],
+  "modules":
+  [
+    {
+      "name":"dave-gnukem",
+      "buildsystem":"autotools",
+      "no-autogen": true,
+      "no-make-install": true,
+      "post-install":
+      [
+        "install -Dvm755 davegnukem -t ${FLATPAK_DEST}/bin/",
+        "install -dv ${FLATPAK_DEST}/share/",
+        "cp -rfv debian/icons/ ${FLATPAK_DEST}/share/",
+        "install -Dm644 debian/desktop/davegnukem.desktop -t ${FLATPAK_DEST}/share/applications/",
+        "install -Dm644 debian/appstream/${FLATPAK_ID}.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/",
+        "install -dv ${FLATPAK_DEST}/share/games/davegnukem/",
+        "cp -rfv gnukem_data-1.0.3/* ${FLATPAK_DEST}/share/games/davegnukem/"
+      ],
+      "sources":[
+        {
+          "type": "archive",
+          "url": "https://github.com/davidjoffe/dave_gnukem/releases/download/1.0.3/davegnukem_1.0.3.orig.tar.xz",
+          "sha256": "77a64e27ab8c006c8cab5cdec005a020dfe383e9aff35748ff9e426c2708d69c"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "sed -i 's#/usr/local#/app#g' Makefile"
```

Comment:
ideal you'd submit a patch to honour PREFIX. It's a matter of adding `?` in the right place. This kind of sed is error prone. And not honour prefix like that is a bug and should be fixed upstream. (assign with `?=`)